AMW_commons/src/main/java/ch/puzzle/itc/mobiliar/common/util/ConfigurationService.java

```diff
@@ -117,7 +135,7 @@ public static String getProperty(ConfigKey key) {
 	 * @param defaultValue
 	 * @return
 	 */
-	public static Integer getPropertyAsInt(ConfigKey key, Integer defaultValue) {
+	protected static Integer getPropertyAsInt(ConfigKey key, Integer defaultValue) {
```

Comment:
I mean, why isn't it public? All other methods that take a key and a defaultValue (e.g. `public static Boolean getPropertyAsBoolean(ConfigKey key, Boolean defaultValue)`) are public.